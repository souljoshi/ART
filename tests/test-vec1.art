void printfln(double x)
{
    printf(x);printc('\n');
}
int main()
{
    vec v1;
    vec v2;
 
    v1 = <10.5, 12.5>;
    v2 = <5.0, 10.0>;

    printfln(v1[0]);
    printfln(v1[1]);
    printfln(v2[0]);
    printfln(v2[1]);

    v1 = v2;

    printfln(v1[0]);
    printfln(v1[1]);
    printfln(v2[0]);
    printfln(v2[1]);
}