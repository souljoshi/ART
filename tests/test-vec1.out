10.500000
12.500000
5.000000
10.000000
5.000000
10.000000
5.000000
10.000000
